net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild

```diff
@@ -3,32 +3,24 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_COMPAT=( python3_{6..9} )
 PYTHON_REQ_USE=( sqlite )
+DISTUTILS_USE_SETUPTOOLS=rdepend
 
 inherit distutils-r1 optfeature
 
 DESCRIPTION="Tool for searching quassel logs from the commandline"
 HOMEPAGE="https://github.com/fish-face/quasselgrep"
 
-if [[ "${PV}" == "9999" ]] ; then
-	EGIT_REPO_URI="https://github.com/65278/quasselgrep.git"
-	inherit git-r3
-	KEYWORDS=""
-else
-	MY_COMMIT=9b6b0bc1252daa6e574363d87d04eebd981215a5
-	SRC_URI="https://github.com/fish-face/${PN}/tarball/${MY_COMMIT} -> ${P}.tar.gz"
-	S="${WORKDIR}/fish-face-${PN}-${MY_COMMIT:0:7}"
-	KEYWORDS="~amd64 ~x86"
-fi
+MY_COMMIT="9b6b0bc1252daa6e574363d87d04eebd981215a5"
+SRC_URI="https://github.com/fish-face/quasselgrep/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${MY_COMMIT}"
+KEYWORDS="~amd64 ~x86"
 
 LICENSE="GPL-2"
 SLOT="0"
 
 RDEPEND="dev-python/future[${PYTHON_USEDEP}]
 	dev-python/pycryptodome[${PYTHON_USEDEP}]
+	dev-python/psycopg:2[${PYTHON_USEDEP}]
```

Comment:
I don't love dragging in PostgreSQL unconditionally. Is there a reason you dropped the optfeature?

IIRC you can use Quassel with other backends.

Reply:
No, I should have left the optfeature, apologies. I have updated it now.